MKTP-968 - Updating SmartApps in Smart Care (Elder Care) category to …

MAblewicz opened the pull request:
…include legal disclaimer

1. SmartApp Name change Elder Care: Daily Routine  -> Smart Care: Daily Routine
2. SmartApp name change Elder Care: Slip & Fall -> Smart Care - Detect Motion
3. Disclaimer page added to both Smart Apps as first page
4. Next Button accepts disclaimer and redirect to app configuration page

Commit:
MKTP-968 - Updating SmartApps in Smart Care (Elder Care) category to include legal disclaimer
1. SmartApp Name change Elder Care: Daily Routine  -> Smart Care: Daily Routine
2. SmartApp name change Elder Care: Slip & Fall -> Smart Care - Detect Motion
3. Disclaimer page added to both Smart Apps as first page
4. Next Button accepts disclaimer and redirect to app configuration page

marstorp reviewed (changes requested):
The default files are intended by tabs, the additional code is mixed spaces and tabs. Replace all spaces with tabs to be consistent throughout.

As far as the implementation I suggest a couple of changes. Keep the disclaimer in the title, this keeps the text black instead of gray, remove the *s around the bold part, it doesn't add any value. Compared to the ticket I think there's one newline added as well that could be removed. It would also be nice if the lines are too long as it simplifies reviews and maintainability.
Also there is a way to make this flow a little bit better by using dynamicPage, with an enum input and the submitOnChange attribute.
I'll share a prototype version with Mariusz Ablewicz through slack.

Commit:
MKTP-968
1. Post review fixes
a) Tabs instead spaces added
2. New Flow implemented
3. Disclaimer text as section
4. Disclaimer lines splitted to make text easier to read

Commit:
1. unnecessary page params removed to avoid confussion

marstorp commented on smartapps/smartthings/smart-care-daily-routine.src/smart-care-daily-routine.groovy:
```diff
@@ -0,0 +1,189 @@
+			section(disclaimerText){
+				input "disclaimerResponse", "enum", title: "Accept terms", required: true,
+						options: ["I agree to these terms", "I do not agree to these terms"],
+						submitOnChange: true
```
Missing a space at the end of the line, after the  `.`

marstorp commented on smartapps/smartthings/smart-care-detect-motion.src/smart-care-detect-motion.groovy:
```diff
@@ -0,0 +1,169 @@
+			}
+		}
+	}
+}
```
Missing a space at the end of the line, after the  `.`

marstorp commented on smartapps/smartthings/smart-care-detect-motion.src/smart-care-detect-motion.groovy:
```diff
@@ -0,0 +1,169 @@
+			}
+		}
+	}
+} 
```
line ends with whitespace

marstorp reviewed (approved):
Just a couple of minor space issues, otherwise looks good.
👍
